tools/compile_tool/main.cpp

```diff
@@ -257,47 +264,107 @@ std::string getFileNameFromPath(const std::string& path,
     }
 }
 
+void printInputAndOutputs(const std::shared_ptr<const ov::Function>& network) {
+    std::cout << "Network inputs:" << std::endl;
+    for (auto&& input : network->inputs()) {
+        std::string layout;
+        if (auto parameter = dynamic_cast<const ov::op::v0::Parameter *>(input.get_node())) {
+            layout = parameter->get_layout().to_string();
+        }
+        std::cout << "    names: {";
+        for (const auto& name : input.get_names()) {
+            if (name != *input.get_names().begin()) {
+                std::cout << ", ";
+            }
+            std::cout << name;
+        }
+        std::cout << "} : " << input.get_element_type() << " / " << layout << std::endl;
+    }
+    std::cout << "Network outputs:" << std::endl;
+    for (auto&& output : network->outputs()) {
+        std::string layout;
+        if (auto result = dynamic_cast<const ov::op::v0::Result *>(output.get_node())) {
+            layout = result->get_layout().to_string();
+        }
+        std::cout << "    names: {";
+        for (const auto& name : output.get_names()) {
+            if (name != *output.get_names().begin()) {
+                std::cout << ", ";
+            }
+            std::cout << name;
+        }
+        std::cout << "} : " << output.get_element_type() << " / " << layout << std::endl;
+    }
+}
+
+
 using TimeDiff = std::chrono::milliseconds;
 
 int main(int argc, char* argv[]) {
     TimeDiff loadNetworkTimeElapsed {0};
 
     try {
-        std::cout << "Inference Engine: " << InferenceEngine::GetInferenceEngineVersion() << std::endl;
+        std::cout << "OpenVINO: " << ov::get_openvino_version() << std::endl;
         std::cout << std::endl;
 
         if (!parseCommandLine(&argc, &argv)) {
```

Comment:
can we also update readme? with information about difference in API and how to use new operation (legacy), tensor names, necessity of network layout if tensor layout is set